app/components/channel-nav/template.hbs

```diff
@@ -1,20 +1,22 @@
-{{#each spaces as |space|}}
-  <li>
-    <h2>{{#link-to "space" space}}{{space.name}}{{/link-to}}</h2>
-    <ul>
-      {{#each space.sortedChannels as |channel|}}
-        <li class="{{if channel.connected "connected" "disconnected"}} {{channel.unreadMessagesClass}} {{if channel.visible "active" ""}}">
-          {{#if channel.isUserChannel}}
-            {{#link-to "space.user-channel" space channel}}{{channel.name}}{{/link-to}}
-          {{else}}
-            {{#link-to "space.channel" space channel}}{{channel.name}}{{/link-to}}
-          {{/if}}
-          <a {{action "leaveChannel" space channel}} class="leave-channel" title="Leave {{channel.name}}">x</a>
+<ul {{did-insert this.didInsertNode}} {{will-destroy this.willDestroyNode}}>
```

Comment:
I think this reads pretty non-descriptive. It should rather say what it actually does, like 
`{{did-insert this.addKeyboardShortcuts}}` and `{{will-destroy this.removeKeyboardShortcuts}}`.

Reply:
I tend to disagree, because the only good function name would be exactly the same as the function name of the keyboard shortcuts library IMO. And then it's confusing which is which.

Reply:
The shortcut library uses `bindKeyboardShortcuts`, my suggestion was `addKeyboardShortcuts`.

Reply:
But "add" doesn't mean anything in this context. "Bind" is the correct term.